src/status_im/ui/screens/wallet/send/views.cljs

```diff
@@ -88,11 +88,7 @@
         immediate-sign-enabled? (or modal? (and sign-enabled? sufficient-funds?))]
     [bottom-buttons/bottom-buttons
      styles/sign-buttons
-     (when sign-enabled?
-       [button/button {:style               components.styles/flex
-                       :on-press            sign-later-handler
-                       :accessibility-label :sign-later-button}
-        (i18n/label :t/transactions-sign-later)])
+     [react/view]
```

Comment:
Why this view?

Reply:
`bottom-buttons` takes positional arguments for left and right button. Since the left button is used for things other than Sign later (e.g. Cancel) I decided to keep it and pass this param for the missing left button. IMO, empty view is more telling than a nil.